packages/@o3r/core/schematics/application/templates/package.json.template

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
```

Comment:
we should detect if we use nx to change that line into `"ng": "nx"`

Reply:
This schematic won't work at all with nx. It's completely out of scope for this PR.

Reply:
https://github.com/AmadeusITGroup/otter/issues/720